x/dex/keeper/limit_order_expiration.go

```diff
@@ -152,5 +152,6 @@ func (k Keeper) PurgeExpiredLimitOrders(ctx sdk.Context, curTime time.Time) {
 		}
 
 		k.RemoveLimitOrderExpirationByKey(ctx, iterator.Key())
+		incExpiredOrders()
```

Comment:
Not sure I understand this metric either. I think we want to track active expiring limit orders. 
Probably have one metric here https://github.com/neutron-org/neutron/blob/feat%2Fdex-telemetry/x/dex/keeper/limit_order_tranche.go#L223
And then decrement where you have the metric above.